ingestion/src/metadata/ingestion/sink/metadata_rest.py

```diff
@@ -145,7 +145,7 @@ def write_tables(self, db_and_table: OMetaDatabaseAndTable):
                 tableType=db_and_table.table.tableType,
                 columns=db_and_table.table.columns,
                 description=db_and_table.table.description,
-                database=db.id,
+                database=db,
```

Comment:
@vivekratnavel make sure this is a entity reference

Reply:
Good catch. Fixed it in the latest patch and verified that sample data ingestion works.